lepidopter-fh/etc/lepidopter-update/version

```diff
@@ -0,0 +1 @@
+1
```

Comment:
@hellais it is not terminated by a new line:
`# cat -A lepidopter-fh/etc/lepidopter-update/version`
`1$`